mifosng-android/src/main/java/com/mifos/mifosxdroid/online/centerdetails/CenterDetailsFragment.java

```diff
@@ -0,0 +1,201 @@
+package com.mifos.mifosxdroid.online.centerdetails;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ProgressBar;
+import android.widget.ScrollView;
+import android.widget.TextView;
+
+import com.mifos.mifosxdroid.R;
+import com.mifos.mifosxdroid.core.MifosBaseActivity;
+import com.mifos.mifosxdroid.core.MifosBaseFragment;
+import com.mifos.objects.group.CenterWithAssociations;
+import com.mifos.utils.Constants;
+import com.mifos.utils.Utils;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * Created by ilya on 12/13/16.
+ */
+
+public class CenterDetailsFragment extends MifosBaseFragment implements CenterDetailsMvpView {
+
+    @BindView(R.id.tv_center_activation_date)
+    TextView activationDate;
```

Comment:
Following [guidelines](https://github.com/openMF/android-client/wiki/Code-style-conventions#51-id-naming) for naming conventions makes it easier for the future developers to work on it.